theme/templates/pages/dashboard.html

```diff
@@ -0,0 +1,321 @@
+            </div>
+            <div class="col-xs-12 col-sm-4 get_started_cell">
+                <a href="https://help.hydroshare.org/introduction-to-hydroshare/getting-started/"
+                   class="get_started_link">Review an example
+                    resource</a>
+            </div>
+            <div class="col-xs-12 col-sm-4 get_started_cell">
+                <a href="https://help.hydroshare.org/introduction-to-hydroshare/getting-started/"
+                   class="get_started_link">What's
+                    new</a>
+            </div>
+        </div>
+
+    </div>
+
+
+    <div>
+        <div class="container dashboard_section_container">
+            <div class="row dashboard_section_container_title_row">
+                <span>
+                 <h3 class="home-page-block-title">Recently Visited </h3>
+                    <span style="float: right">
+                    <a href="#" id="change_me" class="get_started_toggle" data-toggle="collapse"
+                       data-target="#getStarted">Show get started</a>
+                    </span>
+                </span>
+            </div>
+            <table class="table-hover table-striped resource-custom-table" id="recently-visited-resources"
+                   style="width: 100%;">
+                <thead>
+                <tr>
+                    <th>Accessed</th>
+                    <th>Title</th>
+                    <th>First Author</th>
+                    <th>Resource Type</th>
+                    <th>Visibility</th>
+                </tr>
+                </thead>
+                {# ====================================== #}
+                {# == Here comes the data from backend == #}
+                {# ====================================== #}
+                <tbody>
+                {% for r in recent %}
+                    <tr class="data-row">
+                        <td>{{ r.last_accessed | just_in_time }}</td>
+                        <td>
+                            {% autoescape off %}
+                                {% resource_link_builder r.title r.short_id %}
+                            {% endautoescape %}
+                        </td>
+                        <td>{{ r.first_creator }}</td>
+                        <td>
+                            {% autoescape off %}
+                                {% resource_verbose_name_builder r.resource_type %}
+                            {% endautoescape %}
+                        </td>
+                        <td>{% build_privacy_status r.published r.public r.discoverable %}</td>
+                    </tr>
+                {% endfor %}
+
+                </tbody>
+
+            </table>
+        </div>
+
+        <div class="container dashboard_section_container">
+            {% trans "Featured Apps" as whole_title %}
+            {% include "includes/apps_block_title.html" %}
+            <div class="row big-app-row">
+                {% trans "CUAHSI JupyterHub" as title_name %}
```

Comment:
here and everywhere please understand the use of STATIC_URL and apply across entire Pull Request, as appropriate

Reply:
Please explain the repercussions of using that STATIC_URL. Due to code restructuring, this code is obsolete.

Reply:
you have other places, where you are using relative paths to refer to STATIC content, which is wrong. Please address all throughout your frontend code.